package.json

```diff
@@ -72,9 +72,9 @@
     "npm-run-all": "4.1.2",
     "prop-types": "15.6.1",
     "raw-loader": "0.5.1",
-    "react": "16.2.0",
-    "react-dom": "16.2.0",
-    "react-native": "0.50.3",
+    "react": "16.3.1",
```

Comment:
Does the new expo version needs 16.3 explicitly? Isn't it compatible with 16.2?

Reply:
i shall check